lib/matplotlib/colors.py

```diff
@@ -2050,8 +2050,7 @@ def inverse(self, value):
 
 def rgb_to_hsv(arr):
     """
-    Convert float RGB values (in the range [0, 1]), in a numpy array to HSV
-    values.
+    Convert an array of float RGB values (in the range [0, 1]) to HSV values.
```

Comment:
I think this one is fine because of the `np.array` will convert other array-like?  Probably should check...

Reply:
I think that's what the docs say?  (The parameters block immediately below explicitly states array-like, I don't think the first line needs to restate the same info.  Perhaps the top line could even just be "Convert float RGB values to HSV values.")